WebViewControl/GlobalSettings.cs

```diff
@@ -11,7 +12,16 @@ public class GlobalSettings {
         private bool osrEnabled = true;
         private string userAgent;
         private string logFile;
-        private string cachePath = Path.Combine(Path.GetTempPath(), "WebView" + Guid.NewGuid().ToString().Replace("-", null) + DateTime.UtcNow.Ticks);
+        private string cachePath = Path.Combine(Path.GetTempPath(), "WebView" + Guid.NewGuid().ToString().Replace("-", null) + DateTime.UtcNow.Ticks);
+        private Color? backgroundColor;
+
+        public Color? BackgroundColor {
+            get => backgroundColor;
+            set {
+                EnsureNotLoaded(nameof(BackgroundColor));

```

Comment:
@joaompneves in the case of the background color, this check is not needed, right?